### DISCUSSION TITLE
Longhorn volume in degraded state

### DISCUSSION THREAD

**Discussion by btdan** [longhorn/longhorn#4691] [Q&A]
Longhorn 1.3.1 and Ubuntu22.04  ， the disk for longhorn is SSD  disk.
However ,there are about 9 volumes in degraded status. Just as follows:
![1665210955177](https://user-images.githubusercontent.com/5276480/194693561-dd502cbd-a855-4f6a-91c8-eb72da6e77c4.png)
and the detail for the first in the above picture:
![1665211474899](https://user-images.githubusercontent.com/5276480/194693922-d1528617-a7f2-42aa-96bc-19273fc70977.png)

And the Event log is as follows:
![1665211500968](https://user-images.githubusercontent.com/5276480/194693945-e02137da-848b-45ea-89b6-2aeb7f32a42e.png)

I also found this post https://github.com/longhorn/longhorn/discussions/4545
But I have enough disk space for the replicas.
Thanks very much.

**Comment by derekbit** [comment]
Some replicas are at rebuilding state. Probably caused by network issue. Can you provide the support bundle?

**Comment by btdan** [comment]
Thanks very much for your help
I deploy the kubernetes with help of sealos and the network is calico:v3.22.1.
and The tips  in the  terminal window of Ubuntu22.04 is as follows:
![1665214852395](https://user-images.githubusercontent.com/5276480/194696134-19d8b0aa-2aa9-4d93-a420-4fae8889b8d6.png)

Can you tell me how to provide the support bundle? Thanks very much.

**Reply by derekbit** [reply]
You can refer to https://www.suse.com/support/kb/doc/?id=000020145

**Comment by btdan** [comment]
Nearly  Every work node in the kubernetes cluster get the similiar tips
![194696134-19d8b0aa-2aa9-4d93-a420-4fae8889b8d6](https://user-images.githubusercontent.com/5276480/194696192-9122b8b8-1e7f-4430-b767-3febb346102a.png)

But only part of all volume in degrade state.
